options/keepalive.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package options
+
+import (
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/keepalive"
+)
+
+type KeepaliveOption struct {
+	interval time.Duration
+	timeout  time.Duration
+}
+
+func NewKeepaliveOption(interval, timeout time.Duration) KeepaliveOption {
+	return KeepaliveOption{
+		interval: interval,
+		timeout:  timeout,
+	}
+}
+
+func (opt KeepaliveOption) Build() grpc.DialOption {
+	param := keepalive.ClientParameters{
+		PermitWithoutStream: true,
+		Time:                time.Second * 30,
+		Timeout:             time.Second * 5,
```

Comment:
suggestion: Define the `Time` and `Timeout` variables? for example:
```
var (
  time=xxx
  timeout=xxx
)
```

Reply:
ok